internal/cmds/cmds.go

```diff
@@ -388,14 +397,14 @@ func getOutput(ctx *log.Context, stdoutFileName string, stderrFileName string) (
 // checkAndSaveSeqNum checks if the given seqNum is already processed
 // according to the specified seqNumFile and if so, returns true,
 // otherwise saves the given seqNum into seqNumFile returns false.
-func checkAndSaveSeqNum(ctx log.Logger, seq int, mrseqPath string) (shouldExit bool, _ error) {
+func checkAndSaveSeqNum(ctx log.Logger, seq int, mrseqPath string, forceRerun bool) (shouldExit bool, _ error) {
 	ctx.Log("event", "comparing seqnum", "path", mrseqPath)
-	smaller, err := seqnum.IsSmallerThan(mrseqPath, seq)
+	isSavedSeqNumSmallerThanNewSeqNum, err := seqnum.IsSmallerThan(mrseqPath, seq)
```

Comment:
We should trace somewhere (and also put it in an extension event) that the force flag was used.